features_matrices.tex

```diff
@@ -0,0 +1,97 @@
+
+
+SymPy supports matrices with symbolic expressions as elements. There are two
+types of matrices, Mutable and Immutable. Mutable classes are the default in
+SymPy as mutability is important for performance, but it means that standard
+matrices can not interact well with the rest of SymPy. This is because the
+Basic object, from which most SymPy classes inherit, is immutable.
+
+Immutable matrix classes inherit from Basic and can thus interact more
+naturally with the rest of SymPy.
+
+\begin{verbatim}
+In [1]: from sympy import Matrix, symbols, MatrixSymbol
```

Comment:
I see most of the people using python shell `>>>`, so it would be better to use that to maintain uniformity.